functions.php

```diff
@@ -535,16 +535,62 @@ function twentytwentyone_customize_preview_init() {
 add_action( 'customize_preview_init', 'twentytwentyone_customize_preview_init' );
 
 /**
- * Calculate classes for the main <html> element.
+ * Enqueue scripts for the customizer.
  *
  * @since 1.0.0
  *
  * @return void
  */
-function twentytwentyone_the_html_classes() {
+function twentytwentyone_customize_controls_enqueue_scripts() {
+
+	wp_enqueue_script(
+		'twentytwentyone-customize-controls',
+		get_theme_file_uri( '/assets/js/customize.js' ),
+		array( 'customize-base', 'customize-controls', 'underscore', 'jquery' ),
+		get_theme_file_path( 'assets/js/customize.js' ),
+		true
+	);
+
+	$background_color_notice_vars = array(
+		'isDarkMode'                 => twentytwentyone_supports_dark_mode(),
+		'lightPaletteColors'         => array( '#D1E4DD', '#D1DFE4', '#D1D1E4', '#E4D1D1', '#E4DAD1', '#EEEADD', '#FFFFFF' ),
+		'darkModeEnabledMessage'     => esc_html__( 'You currently have dark-mode enabled in your device. Changing the colorpicker will allow you to preview the light mode.', 'twentytwentyone' ),
```

Comment:
```suggestion
		'darkModeEnabledMessage'     => esc_html__( 'You currently have dark mode enabled in your device. Changing the color picker will allow you to preview the light mode.', 'twentytwentyone' ),
```

Just replaced dark-mode with dark mode, colorpicker with color picker.